scvi/train/_trainingplans.py

```diff
@@ -600,13 +600,27 @@ def __init__(
         pyro_module: PyroBaseModuleClass,
         loss_fn: Optional[pyro.infer.ELBO] = None,
         optim: Optional[pyro.optim.PyroOptim] = None,
+        n_obs: Optional[int] = None,
+        optim_kwargs: Optional[dict] = None,
     ):
         super().__init__()
         self.module = pyro_module
         self._n_obs_training = None
 
+        optim_kwargs = optim_kwargs if isinstance(optim_kwargs, dict) else dict()
+        optim_kwargs["lr"] = (
```

Comment:
@adamgayoso I added an option to change lr. It would be inconvenient to write a custom training method just to change lr. WDYT?

Reply:
seems reasonable to me.